### PR TITLE
Initial san francisco and los angeles

### PR DESCRIPTION
**Pull request by balacmi**
This is a merge that adds LA and SF scenarios v1.0

### CODE REVIEW

File: sao_paulo/src/main/java/org/eqasim/sao_paulo/mode_choice/SaoPauloModeAvailability.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
 		// Modes that are always available
 		modes.add(TransportMode.walk);
 		modes.add(TransportMode.pt);
-		modes.add(TransportMode.bike);
+		//modes.add(TransportMode.bike);
```
Comment (sebhoerl) [inline sao_paulo/src/main/java/org/eqasim/sao_paulo/mode_choice/SaoPauloModeAvailability.java]: Is this intended? @balacmi

File: pom.xml [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
-	</modules>
+    <modules>
+        <module>core</module>
+        <module>switzerland</module>
+        <module>ile_de_france</module>
+        <module>sao_paulo</module>
+        <module>automated_vehicles</module>
+        <module>examples</module>
+        <module>san_francisco</module>
+    <module>los_angeles</module>
```
Comment (sebhoerl) [inline pom.xml]: Fix formatting if possible ;)

Comment (sebhoerl) [reply pom.xml]: Ctrl + Shift + F

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # CHANGELOG
 
+- Add option to location assignment to require public transport accessibility
```
Comment (sebhoerl) [inline CHANGELOG.md]: - Remove bike from Sao Paulo
- Add Los Angeles and San Francisco

File: sao_paulo/src/main/java/org/eqasim/sao_paulo/mode_choice/SaoPauloModeAvailability.java (hunk 1 above)
Comment (balacmi) [reply sao_paulo/src/main/java/org/eqasim/sao_paulo/mode_choice/SaoPauloModeAvailability.java]: Yes this is intended as we do not have enough observations in HTS at the moment

File: pom.xml (hunk 2 above)
Comment (balacmi) [reply pom.xml]: I know :P

**Comment by balacmi** [comment]
happy now :)?

**Pull request merged**